toolkit/tools/pkg/imagecustomizerlib/rpmsourcesmounts.go

```diff
@@ -33,11 +36,17 @@ type rpmSourcesMounts struct {
 	allReposConfigFilePath    string
 }
 
-func mountRpmSources(buildDir string, imageChroot *safechroot.Chroot, rpmsSources []string,
+func mountRpmSources(ctx context.Context, buildDir string, imageChroot *safechroot.Chroot, rpmsSources []string,
 	useBaseImageRpmRepos bool,
 ) (*rpmSourcesMounts, error) {
 	var err error
 
+	_, span := otel.GetTracerProvider().Tracer(OtelTracerName).Start(ctx, "mount_rpm_sources")
+	span.SetAttributes(
+		attribute.Int("rpm_sources_count", len(rpmsSources)),
```

Comment:
It would be good to log `useBaseImageRpmRepos` as well.

Reply:
fixed